tools/BaseImageUpdate/baseImage.ps1

```diff
@@ -40,17 +48,28 @@ function Update-ARM-BaseImages
         #>
         [Parameter(Mandatory)]
         [string]
-        $NewASPNetCoreVersion
-    )
+        $NewASPNetCoreVersion,
 
-    Setup-BaseImage-Script
+        <# 
+        Object array contain file paths to base image dockerfile
+        #>
+        [Parameter(Mandatory=$false)]
+        [Object[]]
+        $FileLocale
+    )
 
-    # Get all the Dockerfile file location
-    $fileLocale = $(Get-ChildItem -Recurse -Filter "Dockerfile" | Where {$_.FullName -notlike "*generic-mqtt-tester*"})
+    if ($FileLocale.Count -gt 0)
+    {
+        $fileLocale = $FileLocale
+    }
+    else
+    {
+        $fileLocale = Get-Dockerfile-Locations
+    }
 
     # Replace the underlying ASP .Net Core to the new version
     # Assuming the ARM64 & ARM32 both use the same *-bionic-arm* ASP .Net Core image tag
-    $baseAspNetLocale = $($($fileLocale | Convert-Path) -like "*\base\*" | Resolve-path)
+    $baseAspNetLocale = $($($fileLocale | Convert-Path) -like "*\base\*" -notlike "*\bin\*" | Resolve-path)
```

Comment:
Was this command taking a long time because `bin` was included?

Reply:
I was expecting the `bin` to be deleted when `dotnet clean` is completed. Unfortunately from my experience the `bin` somehow doesn't really get clean. :/